lib/controllers/ServiceBrokerApiController.js

```diff
@@ -23,8 +24,8 @@ class ServiceBrokerApiController extends BaseController {
 
   apiVersion(req, res) {
     /* jshint unused:false */
-    const minVersion = '2.9';
-    const maybeVersion = '2.8';
+    const minVersion = CONST.SF_BROKER_API_VERSION_MIN;
+    const maybeVersion = CONST.SF_BROKER_API_VERSION_MAYBE;
     const minCloudControllerApiVersion = '2.57.0';
     const version = _.get(req.headers, 'x-broker-api-version', '1.0');
     return Promise
```

Comment:
This method checks for CF version. This method still has to be refactored with PlatformManager. . For now can conditionally allow for this to be approved. But please add this as a pending Item in Cross-Consumption story.

Reply:
Created issue #194 . Will update the issue.